### PR TITLE
Keep only COMB

### PR DESCRIPTION
**Pull request by ltkum**
referring to issue #55

### CODE REVIEW

**Comment by ltclm** [comment]
@pakb @ltkum is this ready for the deploy next week?
The tests still need to be modified, but i think they are not executed during the deploy?

**Comment by pakb** [comment]
do you mean to fix the path in this script? I can't find a test that doesn't pass while running `. rc_ci && make all` or `. rc_int && make all`
otherwise yes, this is ready for deploy (with a minor fix from a feedback I have received yesterday with a request out of Switzerland, returning HTTP 500)

**Comment by ltclm** [comment]
@pakb here youre using folders:
https://github.com/geoadmin/service-alti/blob/master/scripts/generate-test-data.sh#L52
which will disappear after the deploy of this code because these data sources are not used anymore.

**Comment by ltclm** [comment]
the same here:
https://github.com/geoadmin/service-alti/blob/master/scripts/generate-test-data.sh#L49

**Comment by pakb** [comment]
is there a new matrix for this new komb dataset? The one used before was still dhm25_25 (or is a matrix still relevant? I don't really understand what it does)

**Comment by ltclm** [comment]
these tests work anymore after the clean-up of the efs:
https://github.com/geoadmin/service-alti/issues/58

The new folders have the same structure as the old ones. 
We have to modfiy the tests and use the new folders in the tests:
- kombo_2m_regio
- kombo_2m_regio_lv95

**Comment by pakb** [comment]
New folders aren't exactly the same, I can't find the tile that was used previously.
Was `1208-4.bt`, I'm looking for `heightservice_1208-4.bt` but the file structure seams a bit different. How can I find the tile?
One thing we could do to check that we are ready, is that we could already clean the EFS for DEV/INT (if it's not already the case). After that if we can run tests on INT, it means we are ready.

**Comment by ltclm** [comment]
the new data has raster data with 5km extent, the filenames have changed.
That's why you wont find the file 1208-4 anymore.
I have already cleaned the efs-dev, tests on dev and int will most probably fail now.

**Comment by pakb** [comment]
I will need some help with that, I don't know how to look for a specific location in these tiles. The area we are looking for is around Interlaken (see https://github.com/geoadmin/service-alti/tree/master/data)

Tests are now failing, that's good-ish 😛

**Comment by ltclm** [comment]
what's the extent of the test region, you will most probably have to load more than one data / raster file.

**Comment by pakb** [comment]
gdalinfo on the old tile outputs :
```
gdalinfo data/swissalti3d/2m/1208-4.bt 
Driver: BT/VTP .bt (Binary Terrain) 1.3 Format
Files: data/swissalti3d/2m/1208-4.bt
Size is 4375, 3000
Coordinate System is:
LOCAL_CS["Unknown",
    UNIT["Meter",1]]
Origin = (628750.000000000000000,176000.000000000000000)
Pixel Size = (2.000000000000000,-2.000000000000000)
Corner Coordinates:
Upper Left  (  628750.000,  176000.000) 
Lower Left  (  628750.000,  170000.000) 
Upper Right (  637500.000,  176000.000) 
Lower Right (  637500.000,  170000.000) 
Center      (  633125.000,  173000.000) 
Band 1 Block=1x3000 Type=Float32, ColorInterp=Undefined
  NoData Value=-32768
  Unit Type: m
```

**Comment by ltclm** [comment]
thats an integration test?

**Comment by pakb** [comment]
integration-ish, yes. This tile is needed because CI as no access to EFS, so we mockup the EFS with only this tile (and made all test land into it).
I have started moving out of this concept with real codebased mockup, but I haven't adapted the whole code base yet.

**Comment by ltclm** [comment]
i have generated the list of the new files covering the old test perimeter of 2m test region:

```bash
$ ogrinfo -ro -al  -spat 628750 170000 637500 176000 index.shp index
INFO: Open of `index.shp'
      using driver `ESRI Shapefile' successful.

Layer name: index
Metadata:
  DBF_DATE_LAST_UPDATE=2020-02-07
Geometry: Polygon
Feature Count: 9
Extent: (443000.000000, 24000.000000) - (895000.000000, 340000.000000)
Layer SRS WKT:
PROJCS["CH1903_LV03",
    GEOGCS["GCS_CH1903",
        DATUM["CH1903",
            SPHEROID["Bessel_1841",6377397.155,299.1528128]],
        PRIMEM["Greenwich",0],
        UNIT["Degree",0.017453292519943295]],
    PROJECTION["Hotine_Oblique_Mercator_Azimuth_Center"],
    PARAMETER["latitude_of_center",46.95240555555556],
    PARAMETER["longitude_of_center",7.439583333333333],
    PARAMETER["azimuth",90],
    PARAMETER["scale_factor",1],
    PARAMETER["false_easting",600000],
    PARAMETER["false_northing",200000],
    PARAMETER["rectified_grid_angle",90],
    UNIT["Meter",1]]
location: String (254.0)
OGRFeature(index):2397
  location (String) = heightservice_2625_1165.bt
  POLYGON ((625000 170000,630000 170000,630000 165000,625000 165000,625000 170000))

OGRFeature(index):2398
  location (String) = heightservice_2625_1170.bt
  POLYGON ((625000 175000,630000 175000,630000 170000,625000 170000,625000 175000))

OGRFeature(index):2399
  location (String) = heightservice_2625_1175.bt
  POLYGON ((625000 180000,630000 180000,630000 175000,625000 175000,625000 180000))

OGRFeature(index):2461
  location (String) = heightservice_2630_1165.bt
  POLYGON ((630000 170000,635000 170000,635000 165000,630000 165000,630000 170000))

OGRFeature(index):2462
  location (String) = heightservice_2630_1170.bt
  POLYGON ((630000 175000,635000 175000,635000 170000,630000 170000,630000 175000))

OGRFeature(index):2463
  location (String) = heightservice_2630_1175.bt
  POLYGON ((630000 180000,635000 180000,635000 175000,630000 175000,630000 180000))

OGRFeature(index):2525
  location (String) = heightservice_2635_1165.bt
  POLYGON ((635000 170000,640000 170000,640000 165000,635000 165000,635000 170000))

OGRFeature(index):2526
  location (String) = heightservice_2635_1170.bt
  POLYGON ((635000 175000,640000 175000,640000 170000,635000 170000,635000 175000))

OGRFeature(index):2527
  location (String) = heightservice_2635_1175.bt
  POLYGON ((635000 180000,640000 180000,640000 175000,635000 175000,635000 180000))
```

it is better to use this command **ogrinfo -ro -al  -spat 628750 170000 637500 176000 index.shp index** for the list of files to pull from efs because the filenames are not stable, they can and will change with a future update.

**Comment by ltclm** [comment]
please also remove the old and deprecated pull sources from the script and github:
/var/local/efs-dev/geodata/bund/swisstopo/swissalti3d/2m
/var/local/efs-dev/geodata/bund/swisstopo/swissalti3d/2m_lv95
/var/local/efs-dev/geodata/bund/swisstopo/swissalti3d/kombo_2m_dhm25
/var/local/efs-dev/geodata/bund/swisstopo/swissalti3d/kombo_2m_dhm25_lv95
/var/local/efs-dev/geodata/bund/swisstopo/dhm25/dhm25_25_matrix
/var/local/efs-dev/geodata/bund/swisstopo/dhm25/dhm25_25_matrix_lv95

File: alti/lib/raster/georaster.py [hunk 1]
```diff
@@ -12,43 +12,31 @@
...
-            'COMB': datapath + 'swissalti3d/kombo_2m_dhm25/index.shp'
-        },
+        21781: datapath + 'swissalti3d/kombo_2m_dhm25/index.shp',
         # LV95
-        2056: {
-            'DTM25': datapath + 'dhm25_25_matrix_lv95/mm0001.shp',
-            'DTM2': datapath + 'swissalti3d/2m_lv95/index.shp',
-            'COMB': datapath + 'swissalti3d/kombo_2m_dhm25_lv95/index.shp'
-        }
+        2056:  datapath + 'swissalti3d/kombo_2m_dhm25_lv95/index.shp'
```
Comment (ltclm) [inline alti/lib/raster/georaster.py]: @ltkum this is the WRONG file path!
kombo_2m_regio
kombo_2m_regio_lv95

**Pull request merged**